ui/lib/kubernetes/addon/engine.js

```diff
@@ -0,0 +1,15 @@
+import Engine from '@ember/engine';
+
+import loadInitializers from 'ember-load-initializers';
+import Resolver from 'ember-resolver';
+
+import config from './config/environment';
+
+const { modulePrefix } = config;
+
+export default class YourEngine extends Engine {
```

Comment:
Let's name this so it's specific 
```suggestion
export default class KubernetesEngine extends Engine {
```

Reply:
Good catch! Darn generator and its generic naming!